packages/block-library/src/gallery/edit.js

```diff
@@ -268,16 +263,21 @@ class GalleryEdit extends Component {
 		const hasImages = !! images.length;
 		const hasImagesWithId = hasImages && some( images, ( { id } ) => id );
 
+		const placeholderText = Platform.select( {
```

Comment:
Could we move this to the top and define it as a constant, to make it easier to notice the differences between mobile and web?

Reply:
Good suggestion :+1: . Updated.